src/ol/layer/layer.js

```diff
@@ -152,7 +152,7 @@ ol.layer.Layer.prototype.handleSourcePropertyChange_ = function() {
  * its layers collection, layer filters in {@link ol.Map#forEachLayerAtPixel}
  * will not filter the layer, and it will be rendered on top. This is useful for
  * temporary layers. To remove an unmanaged layer from the map, use
- * `#setMap(null)`.
+ * `#setMap(null)`. Unmanaged layers behave as if their Z-index is 1000.
```

Comment:
Does this mean a user is able to place some of his managed layers above unmanaged layers?